config.yaml

```
@@ -28,13 +32,12 @@ serving:
   maxConcurrentModels: 2
   grpcConfigTimeout: 10 # timeout in seconds
   grpcPredictTimeout: 60
-  metricsPath: "/monitoring/prometheus/metrics"
 
 proxy:
   replicasPerModel: 3
   grpcTimeout: 10
 
-serviceDiscovery:
+  serviceDiscovery:
```

Too much indentation
```suggestion
serviceDiscovery:
```

---

done